src/platform/forms/tests/save-in-progress/ApplicationStatus.unit.spec.jsx

```diff
@@ -113,9 +108,7 @@ describe('schemaform <ApplicationStatus>', () => {
       />,
     );
     expect(tree.subTree('.usa-alert-warning')).to.not.be.false;
-    expect(tree.subTree('.usa-button-primary').text()).to.equal(
-      'Start a new application',
-    );
+    expect(tree.text()).to.include('start a new application');
```

Comment:
I'm not sure why this passes now; AFAIK, `include` isn't a case-insensitive assertion. Does it fail if it's "Start"?

Reply:
Oh, I see now it's for the tree text, not the button text.